docs/testrunner-toolkit/faqs.md

```diff
@@ -56,3 +56,29 @@ Yes! You can find them here: [Automation Framework Examples](testrunner-toolkit/
 The GitHub repository includes CI/CD examples of `saucectl` using:
 * [GitHub Actions Workflows](https://help.github.com/en/actions) 
 * [CircleCI Pipelines](https://circleci.com/docs/2.0/configuration-reference/) 
+
+## How Do I Use a Proxy ?
+
+If you need to go through a proxy server, you can set it through the following variables.
+- `HTTP_PROXY`: Proxy to use to access HTTP websites
+- `HTTPS_PROXY`: Proxy to use to access HTTPS websites
+
+SauceCtl reaches Sauce Labs infrastructure to get latest docker image available or upload the test package to Sauce Cloud.
+When running in docker-mode, the docker container needs to access the tested website and Sauce Labs to upload results.
+
+Therefore, you may be required to set twice the proxy settings.
```

Comment:
Hi @FriggaHel -- Are you saying here that, even if you are testing locally using Docker, you may still have to set both an `HTTP` proxy _and_ an `HTTPS` proxy because `saucectl` needs to connect to Sauce Labs (HTTPS) to get the latest image? Thanks for clarification -- I am just trying to understand what you mean when you say "set the proxy settings twice" -- you mean once for HTTP and once for HTTPS, as shown in your example, correct?

Reply:
@sweeneyskirt-sl Sorry that it was not that clear.

1st setting (effective on `saucectl` calls to Sauce Labs infrastructure):
```
PS> $Env:HTTP_PROXY=http://my.proxy.org:3128/
PS> $Env:HTTPS_PROXY=http://my.proxy.org:3128/
```

2nd setting (effective inside docker image):
```
PS> saucectl run -e HTTP_PROXY=${Env:HTTP_PROXY} -e HTTPS_PROXY=${Env:HTTPS_PROXY} --test-env docker
```

In Sauce Cloud mode, it could be as simple as that:
```
PS> $Env:HTTP_PROXY=http://my.proxy.org:3128/ # This should be facultative too.
PS> $Env:HTTPS_PROXY=http://my.proxy.org:3128/
PS> saucectl run --test-env sauce
```

Reply:
Okay. Thanks for clarifying.